config/monitoring/prometheus/monitor.yaml

```diff
@@ -13,7 +13,10 @@ spec:
     - path: /metrics
       port: https # Ensure this is the name of the port that exposes HTTPS metrics
       scheme: https
-      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      authorization:
```

Comment:
I'd say we should just remove this file which creates YET another serviceMonitor based on direct token

since we already have correct serviceMonitor based on that SA : https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build-service/base/monitoring.yaml#L27